src/towncrier/test/test_project.py

```diff
@@ -39,6 +39,46 @@ def test_tuple(self):
         version = get_version(temp, "mytestproja")
         self.assertEqual(version, "1.3.12")
 
+    def test_import_fails(self):
+        """
+        The project cannot be found.
+        """
+        if sys.version_info >= (3, 6):
+            expected_exception = ModuleNotFoundError
+        else:
+            expected_exception = ImportError
+
+        with self.assertRaises(expected_exception):
+            get_version(".", "mytestproj_import_fails")
+
+    def test_already_installed_import(self):
+        """
+        An already installed package will be checked before cwd-found packages.
+        """
+        project_name = "mytestproj_already_installed_import"
+
+        temp = self.mktemp()
+        os.makedirs(temp)
+        os.makedirs(os.path.join(temp, project_name))
+
+        with open(os.path.join(temp, project_name, "__init__.py"), "w") as f:
+            f.write("__version__ = (1, 3, 12)")
```

Comment:
```suggestion
            f.write("__version__ = "version.on.disk")
```

but maybe we don't need at all.

Reply:
I'm not clear what this would clarify.  `:]`

Reply:
'1.3.12' vs '1.3.13` is a tiny difference :) .. .but maybe we don't even need to create these files ... and only have `sys_path_temp`

Reply:
This is testing that `get_version()` get's the version from the proper place.  Unless we extracted the selecting of the package to be a separate activity (sure, that sounds maybe good) then don't we need to put versions into the packages so we can make sure the proper one was used?

'version on disk' doesn't seem to really distinguish between the two, but 'installed' vs. 'source' might?  Though seems reasonable to keep them as 'versions' instead of 'names'.  Perhaps `(1, 3, 12)` and `(2, 1, 5)` would make them usefully distinct so the difference is harder to overlook?